.github/workflows/deploy.yml

```diff
@@ -41,15 +41,14 @@ jobs:
         uses: actions/cache@v2
         with:
           path: |
-            ~/.npm
             ./frontend/node_modules
-          key: npm-build-${{env.NODE_VERSION}}-${{ hashFiles('client/package-lock.json', 'client/package.json') }}
-      - name: Node setup
+          key: npm-build-${{env.NODE_VERSION}}-${{ hashFiles('frontend/yarn.lock', 'frontend/package.json') }}
+      - name: Install Dependencies
         working-directory: ./frontend
-        run: npm install
+        run: yarn
```

Comment:
I observe this is `yarn` here but `yarn install` in the next one, and I'm insufficiently confident in my yarn knowledge to know if they're equivalent or not.

Reply:
`yarn` and `yarn install` are equivalent